### DISCUSSION TITLE
Trouble Installing from Source on Windows 11

### DISCUSSION THREAD

**Discussion by jzhang1705** [rdkit/rdkit#6841] [Q&A]
I have been having trouble building RDKit from source on Windows 11, the steps are listed here:   https://www.rdkit.org/docs/Install.html#installation-from-source 

I am stuck because every time I run “cmake ..” in my RDkit build directory, I get this error message: 

```
CMake Error at C:/Program Files/CMake/share/cmake-3.28/Modules/FindPackageHandleStandardArgs.cmake:230 (message): 

  Could NOT find Boost (missing: python) (found suitable version "1.83.0", 

  minimum required is "1.58.0") 

Call Stack (most recent call first): 

  C:/Program Files/CMake/share/cmake-3.28/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE) 

  C:/Program Files/CMake/share/cmake-3.28/Modules/FindBoost.cmake:2392 (find_package_handle_standard_args) 

  CMakeLists.txt:328 (find_package) 

```

I can't seem to find C:\RDKit\lib and C:\boost\lib so I was unable to put it on my path. I have also tried changing my Python path configurations.

**Comment by kienerj** [comment]
Maybe you can find the solution how to build it in below previous discussion:

https://github.com/rdkit/rdkit/discussions/6148

(note: this also works for windows 10)

**Reply by jzhang1705** [reply]
Will this allow me to write code on top of the current RDkit C++ files? The instructions for building from conda suggest we're building from a different repository.

**Comment by jzhang1705** [comment]
I guess to clarify, I want to use ONLY the C++ functionalities of RDKit. I do not want to use Python at all. Is it possible to do this?
